Kiosk/Auction Listings/ListingsViewModel.swift

```diff
@@ -208,11 +208,13 @@ class ListingsViewModel: NSObject, ListingsViewModelType {
         #endif
     }
 
-    private class func DefaultScheduler<T>(onBackground background: Bool)(observable: Observable<T>) -> Observable<T> {
-        if background {
-            return observable.observeOn(backgroundScheduler)
-        } else {
-            return observable.observeOn(MainScheduler.instance)
+    private class func DefaultScheduler<T>(onBackground background: Bool) -> (observable: Observable<T>) -> Observable<T> {
```

Comment:
Yeah, I was miffed when they deprecated the hold one but it's a lot more legible.